storage/client/queries.go

```diff
@@ -534,3 +534,15 @@ func (qf QueryFactory) TxVolumesQuery() string {
 		OFFSET $3::bigint
 	`
 }
+
+func (qf QueryFactory) DailyActiveAccountsQuery() string {
+	return `
+		SELECT window_end - INTERVAL '1 DAY' AS bucket_start, active_accounts
```

Comment:
This `window_end - INTERVAL '1 DAY'` part could be avoided if the analyzer backend and the API used unified representation for buckets  

BUT:
- it made more sense to define buckets with `window_end` (instead of `window_start`) within the analyzer backend code (personally I think it would make the analyzer code a bit less readable if window_start would be used)
- on the other hand, the existing stats API (`stats/tx_volume`) defines buckets with start time already, and it is nice to use the unified API for `stats/active_accounts` as well
- since API and the analyzer are nicely decoupled already, I think that using a separate representation and handling the conversion here is ok/better than alternative of updating the analyzer code.

Reply:
Agreed on all points, will update.